### PR TITLE
Emergency Alert

### PR DESCRIPTION
**Pull request by mcherie**
# Emergency Alert

Screen Shot 2022-11-09 at 11 56 26 AM

Fixes Line 22 on the spreadsheet

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

- [ ] My code follows the style guidelines of this project at all breakpoints
- [ ] This matches all the required acceptance criteria
- [ ] This code has been tested in Chrome & Safari

### CODE REVIEW

File: adobe-ourwork-aem-dev/components/EmergencyAlert/EmergencyAlert.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import '../../styles/lumen-tokens';
+
+.emergency-alert {
+  background: $blue-nataliya;
```
Comment (mcherie) [inline adobe-ourwork-aem-dev/components/EmergencyAlert/EmergencyAlert.scss]: naming ^

File: adobe-ourwork-aem-dev/components/EmergencyAlert/EmergencyAlert.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  }
+
+  &__body {
+    @include typographylargeparagraphsmallcopy_regular;
+    color: $white;
+  }
+
+  &__button {
+    @include typographysmallparagraphbodycopy_regular;
+    background: $blue-dark-nataliya;
```
Comment (mcherie) [inline adobe-ourwork-aem-dev/components/EmergencyAlert/EmergencyAlert.scss]: naming ^

**Comment by nataliya-ioffe** [comment]
@mcherie

**Review by e-schultz — APPROVED** [review]
✅  looks good

**Pull request merged**